driver/remote/driver.go

```diff
@@ -78,12 +85,18 @@ func (d *Driver) Rm(ctx context.Context, force, rmVolume, rmDaemon bool) error {
 }
 
 func (d *Driver) Client(ctx context.Context, opts ...client.ClientOpt) (*client.Client, error) {
-	opts = append([]client.ClientOpt{
-		client.WithContextDialer(func(ctx context.Context, _ string) (net.Conn, error) {
-			return d.Dial(ctx)
-		}),
-	}, opts...)
-	return client.New(ctx, "", opts...)
+	d.clientOnce.Do(func() {
+		opts = append([]client.ClientOpt{
+			client.WithContextDialer(func(ctx context.Context, _ string) (net.Conn, error) {
+				return d.Dial(ctx)
+			}),
+			client.WithTracerDelegate(delegated.DefaultExporter),
```

Comment:
This is pretty 😬 , normally called by `DriverHandle`. Mostly weird as other drivers don't need to call it.

Reply:
But do not `client.WithTracerDelegate` already set as opt in https://github.com/docker/buildx/blob/cfef22ddf0ccff52a8aced77c6115249d78b914b/driver/remote/driver.go#L80 via https://github.com/docker/buildx/blob/cfef22ddf0ccff52a8aced77c6115249d78b914b/driver/manager.go#L167

Reply:
This is how it works in regular mode and for other drivers. But for remote driver the call path can be that `Info()` is called that internally calls `Client()` (and that client is now cached). In that case no delegate is set up.

Maybe a possible solution would be to move `ClientOpt` as part of `InitConfig`

Reply:
> Maybe a possible solution would be to move `ClientOpt` as part of `InitConfig`

Ah right yeah that would be neat

Reply:
This still would mean that every driver would need to handle the `ClientOpt` and if they don't then at least the tracing would not work.